ui/src/components/EntityTable/EntityTable.jsx

```diff
@@ -42,6 +42,8 @@ class EntityTable extends Component {
     this.state = {
       result: props.result
     };
+
+    this.getHighlight = this.getHighlight.bind(this);
```

Comment:
No need to bind class methods; instead use I suggest to use `class properties` to attach persist the context.
For more details [see](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties)
Note: By default our babel configuration supports this plugin